src/llm/http_llm_calculator.cc

```diff
@@ -389,6 +389,25 @@ static std::string packIntoServerSideEventMessage(const std::string& message) {
     return ss.str();
 }
 
+static std::string packPromptTokens(const ov::Tensor& input) {
+    std::stringstream ss;
+    ss << "prompt_token_ids: [";
+    if (input.get_element_type() != ov::element::i64) {
+        ss << "Warning: expected ov::element::i64 and got " << input.get_element_type() << " for input tokens.]";
```

Comment:
That's quite misleading. We will print such warning, but what happens with the execution? And user will get it on trace level only. I think we should either support other types and make more generic method for printing values or say more explicitly like `"Cannot print input tokens - expected input type ov::element::i64 and got " << input.get_element_type()"`